<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OcrForm</class>
 <widget class="QWidget" name="OcrForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,15">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>图片路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Path">
         <property name="text">
          <string>image file name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_OpenFile">
         <property name="text">
          <string>选择图片</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>显示图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Ocr">
             <property name="text">
              <string>识别图片</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>375</width>
              <height>275</height>
             </rect>
            </property>
            <widget class="QLabel" name="label_Image">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>-10</y>
               <width>377</width>
               <height>201</height>
              </rect>
             </property>
             <property name="text">
              <string>image</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,15">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>图片文字</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_Result"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusBar">
       <property name="text">
        <string>状态信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OpenFile</sender>
   <signal>clicked()</signal>
   <receiver>OcrForm</receiver>
   <slot>pushButton_OpenFile_OnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Ocr</sender>
   <signal>clicked()</signal>
   <receiver>OcrForm</receiver>
   <slot>pushButton_Ocr_OnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButton_Ocr_OnClick()</slot>
  <slot>pushButton_OpenFile_OnClick()</slot>
 </slots>
</ui>
